benefits/enrollment/templates/enrollment/index.html

```diff
@@ -12,9 +12,13 @@
 {% block paragraph_content %}
   {{ block.super }}
   {% if payment_processor %}
-    <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>
-      {{ payment_processor.loading_text }}
-    </button>
+    <div class="row justify-content-center justify-content-lg-end">
```

Comment:
I added alignment classes on this page for testing purposes. I wanted to make sure any `<form>` or `button` CSS changes weren't causing any regressions on this page, and that the same button CSS + classes can be used to get the desired design.

Reply:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/194445249-1ffe5d63-6aa1-4313-bcd9-eb9dcc6dbfee.png">